Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/ConceptIDLookup.java

```diff
@@ -45,6 +46,9 @@
 	@Autowired
 	private XMLMessageFactory messageFactory;
 
+    @Autowired
+    private IIndexService indexService;
+
```

Comment:
This looks like a formatting issue because the existing lines are using tab spacing.